src/pages/upload/index.tsx

```diff
@@ -41,7 +41,7 @@ const Upload: React.FC = () => {
         //if we set this to `false` we can list all the files and
         //call `uploaderRef.current.uploadFile()` when we are ready
         autoUpload={true}
-        signingUrl={viewer.s3_signing_url}
+        signingUrl={`/api/aws/sign-s3`}
```

Comment:
this could be moved over to the API in `egghead-rails` instead of hard coded here

Reply:
What would be the benefit of having `egghead-rails` provide the signing route?

One drawback I see is that if we ever wanted to change the route, we'd have to deploy changes to two systems.